app/src/userpages/components/StreamPage/Show/HistoryView/index.jsx

```diff
@@ -271,8 +271,9 @@ class HistoryView extends Component<Props, State> {
                                 disabled={disabled}
                             />
                             <Button
+                                type="secondary"
+                                size="mini"
```

Comment:
I don't think these should be `mini`? 
https://share.goabstract.com/a4e056bf-04d3-431e-b018-670016445b67?mode=build&sha=1e7a90aef062fa93d4f82b6124f82c128e7d7e52